.gitignore

```diff
@@ -130,5 +129,4 @@ ENV/
 
 # data
 .data/
-datasets/
```

Comment:
add this back

Reply:
mnist.py is in there, it will get ignored. 
the gitignore already contains "mnist/", is that not enought? I think we don't use any other datasets in test. where should I move mnist.py then?

Reply:
so what about moving the mnist.py to model directly... I have usually associated `datasets/` with a folder with stored datasets

Reply:
ok done